Classes/Controller/FeUsersController.php

```diff
@@ -117,7 +116,9 @@ public function renderAjax(ServerRequestInterface $request): Response
 			if (isset($row['usergroup'])) {
 				$usergroups = [];
 				foreach (explode(',', $row['usergroup']) as $usergroupUid) {
-					$usergroups[] = $this->getUsergroupNameByUid($usergroupUid);
+					if (!empty($usergroupUid)) {
+						$usergroups[] = $this->getUsergroupNameByUid($usergroupUid);
+					}
 				}
 				$row['usergroup'] = implode(', ', $usergroups);
 			}
```

Comment:
I would recommend simplifying this further to avoid the nesting levels and improve readability to something like this:

```
$usergroups = array_filter(
    array_map(
        function (string|int $usergroupUid): string {
            $usergroupUid = (int)$usergroupUid;

            return $usergroupUid ? $this->getUsergroupNameByUid($usergroupUid) : null;
        },
        explode(',', $row['usergroup'] ?? '')
    )
);
```

Completely untested and written entirely in the Gitlab comment box, so definitely look it over and test before use! Could perhaps to with deconstructing further, but it will ensure the values are typed and filtered correctly

Reply:
Since atach is php 7.4 union typing won't work here. Implementation working, cheers